websites/A/Apple/presence.ts

```diff
@@ -73,6 +74,8 @@ presence.on("UpdateData", async () => {
       largeImageKey: logoArr[setting.logo] || "logo"
     };
 
+    let showElapsed = true;
```

Comment:
wow that's the most unecessary thing that I ever seen. Please do it in different way